Dockerfile

```diff
@@ -8,6 +8,7 @@
 FROM  openjdk:8-jdk-alpine
 
 ENV SCALA_VERSION 2.11.11
+ENV MESOS_ACTOR_VERSION 0.0.4
```

Comment:
Maybe using the one from the ENV could help, thinking durning bulid release cycle if you wanted to be able to test a new snapshot, e.g.:
`ENV MESOS_ACTOR_VERSION=${MESOS_ACTOR_VERSION:-0.0.4}`

Reply:
I don't think that is valid syntax in Dockerfile? this is used for the ADD below, but maybe I can instead use a wildcard to avoid having version in the Dockerfile at all...